Lottie.Forms/Platforms/Ios/AnimationViewRenderer.cs

```diff
@@ -131,19 +132,28 @@ private void InitAnimationViewForElement(AnimationView theElement)
                 _animationView.PlayWithCompletion(PlaybackFinishedIfActually);
             }
 
-            if (_animationView != null)
-            {
-                SetNativeControl(_animationView);
-                SetNeedsLayout();
-            }
+            SetNativeControl(_animationView);
```

Comment:
_animationView is already initialized at the top of the method. Why check for the null case?